engine/gui/src/gui.cpp

```diff
@@ -2519,6 +2519,9 @@ namespace dmGui
         }
         if (n->m_Node.m_Text)
             free((void*)n->m_Node.m_Text);
+        if (n->m_Node.m_HasResetPoint) {
+            free(n->m_Node.m_ResetPointProperties);
+        }
```

Comment:
No need for the if-statement, free() can handle a null pointer.

Reply:
fixed, thank you